app/screens/StateScreen.tsx

```diff
@@ -39,6 +50,72 @@ export function StateScreen() {
     }))
   }
 
+  const createSnapshot = () => {
+    if (!activeTab) {
+      console.log("No active client to create snapshot from")
+      return
+    }
```

Comment:
Prefer one-liner guards if possible.

```suggestion
    if (!activeTab) return console.log("No active client to create snapshot from")
```